docs/content/commands/npm-pkg.md

```diff
@@ -76,6 +76,14 @@ Returned values are always in **json** format.
     npm pkg get contributors[0].email
     ```
 
+    For complex fields you can also use text indices in square brackets
+    to specifically select a child field. This is especially helpful
+    with the exports object:
+
+    ```bash
+    npm pkg get "exports[.].require"
```

Comment:
"text indices" suggests like, 0, 1, 2, etc. is `.` special?

Reply:
No '.' is a valid ascii character hence "text"

Reply:
right but "indices" are only numeric. if it's object properties then "properties" might be a better word?

Reply:
```
    For complex fields you can also name a property in square brackets
    to specifically select a child field. This is especially helpful
    with the exports object:
```

Reply:
Sounds great to me, thanks!